lib/engine/game/g_22_mars/step/select_or_discard.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+module Engine
+  module Game
+    module G22Mars
+      module Step
+        class SelectOrDiscard < Engine::Step::SimpleDraft
+          ACTIONS = %w[bid pass].freeze
+
+          def setup
+            @companies = @game.companies
+          end
+
+          def available
+            [@companies.first]
+          end
+
+          def finished?
+            @companies.none?
```

Comment:
`Array.none?` shouldn't be used unless you are passing it a block. Use `Array.empty?` instead.
```suggestion
            @companies.empty?
```